README.md

```diff
@@ -9,41 +9,42 @@
 The library provides easier access to some of the functions provided by OpenSSL.
 It does not attempt to wrap its data types but adds some functionality for
 file access, file protection, configuration handling, HTTP and TLS connections,
-certificate verfication and status checking with CRLs and/or OCSP, and logging.
-The Unified Trust Anchor (UTA) library can be used for enhanced file protection.
+certificate verification and status checking with CRLs and/or OCSP, and logging.
+The [Unified Trust Anchor (UTA)](../../../libuta/) library
```

Comment:
```suggestion
The [Unified Trust Anchor (UTA)](https://github.com/siemens/libuta/) library
```

Reply:
Generally I prefer relative links as far as possible because they are more stable in case the whole structure (e.g., base directory or website) is moved.
The above relative link does work, though it is a little counter-intuitive because GitHub automatically inserts `blob/<branch>/`.
Yet there are also advantages when using absolute URLs. 
BTW, there is already an absolute URL in `README.md` referencing the UTA lib.
Sure to prefer absolute URLs in the above line and in line 82?

Reply:
Yes, because forking or uploading to a different service (e.g. GitLab) otherwise breaks the links.

Reply:
Good point - -done.